src/components/listing-search-page/Card/card.style.js

```diff
@@ -0,0 +1,105 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const CardContainer = styled.div`
+  border-radius: 0.5rem;
+  border: 0.0625rem solid ${colors.midGrey};
+  display: flex;
+  flex-direction: column;
+  margin: 2rem;
+  overflow: hidden;
+
+  ${mq.tabletWide`
+      flex-direction: row;
+  `}
+`;
+
+export const CardImageContainer = styled.div`
+  max-height: 8.5rem;
+  overflow: hidden;
+`;
+
+export const ImageContainer = styled.div`
+  overflow: hidden;
+  position: relative;
+
+  ${mq.mobileWide`
+    max-height: 15rem;
+  `}
+
+  ${mq.tablet`
+    max-height: 20rem;
+  `}
+`;
+
+export const Badge = styled.div`
+  background-color: ${colors.green};
+  border-radius: 0.3125rem;
+  color: ${colors.white};
+  margin: 1rem;
+  padding: 0.3rem 1.2rem;
+  position: absolute;
+
+  ${mq.tablet`
+      margin: 0.5rem;
+  `}
+`;
+
+export const Image = styled.img`
+  min-height: 100%;
+  min-width: 100%;
+`;
+
+export const CardText = styled.div`
+  padding-left: 1rem;
+`;
+
+export const PriceText = styled.h4`
+  ${mq.tabletWide`
+      font-size:1.125rem;
+      font-weight: 500;
+      margin:1rem 0;
```

Comment:
```suggestion
    font-size:1.125rem;
    font-weight: 500;
    margin:1rem 0;
```

fix the indentation here.